dataland-frontend/src/components/pages/QualityAssurance.vue

```diff
@@ -96,105 +165,154 @@ export default defineComponent({
-     * Gathers meta and company information associated with a dataset if the information can be retrieved
-     * @param dataId the ID of the corresponding dataset
-     * @returns a promise on the fetched data object
-     */
-    async addDatasetAssociatedInformationToDisplayList(dataId: string): Promise<QaDataObject> {
-      const metaDataResponse = await assertDefined(
-        this.apiClientProvider
-      ).backendClients.metaDataController.getDataMetaInfo(dataId);
-      this.metaInformation = metaDataResponse.data;
-      const companyResponse = await assertDefined(
-        this.apiClientProvider
-      ).backendClients.companyDataController.getCompanyById(this.metaInformation.companyId);
-      this.companyInformation = companyResponse.data.companyInformation;
-      return {
-        dataId: dataId,
-        metaInformation: this.metaInformation,
-        companyInformation: this.companyInformation,
-      };
-    },
     /**
      * Navigates to the view framework data page on a click on the row of the company
      * @param event the row click event
      * @returns the promise of the router push action
      */
     goToQaViewPage(event: DataTableRowClickEvent) {
-      const qaDataObject = event.data as QaDataObject;
-      const qaUri = `/companies/${qaDataObject.metaInformation.companyId}/frameworks/${qaDataObject.metaInformation.dataType}/${qaDataObject.dataId}`;
+      const qaDataObject = event.data as ReviewQueueResponse;
+      const qaUri = `/companies/${qaDataObject.companyId}/frameworks/${qaDataObject.framework}/${qaDataObject.dataId}`;
       return this.$router.push(qaUri);
     },
+
     /**
-     * Updates the data for the current page
-     * @param event event containing the new page
+     * Resets selected frameworks and searchBarInput
      */
-    async onPage(event: DataTablePageEvent) {
-      this.currentPage = event.page;
-      await this.getQaDataForCurrentPage();
+    resetFilterAndSearchBar() {
+      this.currentChunkIndex = 0;
+      this.selectedFrameworks = [];
+      this.availableReportingPeriods = [];
+      this.searchBarInput = '';
+    },
+    /**
+     * Updates the current Page
+     * @param event DataTablePageEvent
+     */
+    onPage(event: DataTablePageEvent) {
+      window.scrollTo(0, 0);
+      if (event.page != this.currentChunkIndex) {
+        this.currentChunkIndex = event.page;
+        this.firstRowIndex = this.currentChunkIndex * this.datasetsPerPage;
+        this.getQaDataForCurrentPage();
+      }
+    },
+  },
+  computed: {
+    numberOfUnreviewedDatasets(): string {
+      if (!this.waitingForData) {
+        if (this.totalRecords === 0) {
+          return 'No results for this search.';
+        } else {
+          const startIndex = this.currentChunkIndex * this.datasetsPerPage + 1;
+          const endIndex = Math.min(startIndex + this.datasetsPerPage - 1, this.totalRecords);
+          return `Showing results ${startIndex}-${endIndex} of ${this.totalRecords}.`;
```

Comment:
Ist Erbsenzählerei, aber das Format ist unter COMPANIES etwas anders.

COMPANIES: 1-100 of X results
QA: Showing results 1-10 of X.

Würde ich vereinheitlichen (außer dem 10/100 Ding).

Reply:
So wie das hier ist, ist es aber konsistent zu dem "All Requests Panel". Seh aber den Punkt, dass das am besten überall gleich sein sollte. Persönlich finde ich das Showing results schöner, habe da aber keine sehr starke Meinung zu. Wie sieht das bei dir und @emanueldfine aus?

Reply:
Das stimmt, dann kann das so bleiben. Idealerweise würde man dann stattdessen COMPANIES anpassen.

Reply:
Ist für mich die Frage ob man das in der Story hier macht oder irgendwann wenn man die Companies Page anpasst.

Reply:
Das stimmt.